drivers/dma/dmamux_stm32.c

```diff
@@ -273,26 +349,50 @@ static const struct dma_driver_api dma_funcs = {
 #define DEV_DMA2 COND_CODE_1(DT_NODE_HAS_STATUS(DT_NODELABEL(dma2), okay), \
 			     DEVICE_DT_GET(DT_NODELABEL(dma2)), NULL)
 
+#define BDMA_1_BEGIN_DMAMUX_CHANNEL DT_PROP_OR(DT_NODELABEL(bdma1), dma_offset, 0)
```

Comment:
Nit: Please update upper comment which specifies " valid for series having 1 dmamux and 1 or 2 dmas."

Reply:
Fixed